example/storage/simple_postgres_repository.go

```diff
@@ -32,23 +32,24 @@ func NewSimpleAnonymizingPostcardRepository(ctx context.Context, db *sql.DB) (sq
 		ctx,
 		db,
 		sql2.NewPostgresSchemaAdapter[*postcard.Postcard]("PostcardSimpleAnonymizing"),
-		sql2.NewSimpleSerializer(
-			sql2.NewAnonymizingMarshaler(
+		sql2.NewAESAnonymizingSerializer[*postcard.Postcard](
```

Comment:
Don't we want to move both the serializer and marshaler to the new package as well? I guess it is quite intuitive to have implementations of the interface in different packages. Also, are the serializer and marshaler useful for SQL repositories only? I do not think so.

Reply:
I guess this applies to some other components also. All the JSONMarshaler, EventMapper, and SimpleSerializer are schema and event database agnostic, don't they?

Reply:
Good point, I'll move them in another PR